media/admin/com_virtuemart/models/zasilkovna_src/VirtueMartModelZasilkovna/Order/Detail.php

```diff
@@ -88,8 +102,20 @@ public function validateFormData(array $formData)
                 );
             }
         }
+    }
 
-        return $validationReport;
+    private function validateMinimumValues(array $formData, array $fields, DetailFormValidationReport $validationReport)
+    {
+        foreach ($fields as $field => $translationKey) {
+            if (isset($formData[$field]) && $formData[$field] < 0) {
```

Comment:
Neznamená to, že 0 projde? To by nesedělo s hláškou.

Reply:
Opraveno.